edpm_ansible/roles/edpm_nftables/defaults/main.yml

```diff
@@ -0,0 +1,22 @@
+---
+# Copyright 2022 Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+# All variables intended for modification should be placed in this file.
+
+# All variables within this role should have a prefix of "edpm_nftables_"
+edpm_nftables_src: /var/lib/edpm-config/firewall
+edpm_nftables_default_rules_prefix: EDPM
```

Comment:
nitpicking: it's a chain_prefix, not a rule :)

Reply:
ouch, my bad.